lib/cli-engine.js

```diff
@@ -77,28 +77,7 @@ var DEFAULT_PARSER = require("../conf/eslint.json").parser;
 // Private
 //------------------------------------------------------------------------------
 
-var defaultOptions = {
-    configFile: null,
-    baseConfig: false,
-    rulePaths: [],
-    useEslintrc: true,
-    envs: [],
-    globals: [],
-    rules: {},
-    extensions: [".js"],
-    ignore: true,
-    ignorePath: null,
-    parser: DEFAULT_PARSER,
-    cache: false,
-    // in order to honor the cacheFile option if specified
-    // this option should not have a default value otherwise
-    // it will always be used
-    cacheLocation: "",
-    cacheFile: ".eslintcache",
-    fix: false,
-    allowInlineConfig: true,
-    cwd: process.cwd()
-};
+defaultOptions = assign({}, defaultOptions, {cwd: process.cwd()});
```

Comment:
Not sure I understand why this is necessary. Default is already set to `process.cwd` why do you need to reassign it again?